### PR TITLE
Added links to simple Golang wiki, pastebin and url shortener apps

### PR DESCRIPTION
**Pull request by prologic**
These are somple simple easy to use and deploy self-hosted apps written in
Goland for performance and efficiency.

### CODE REVIEW

**Comment by danger-awesome-Selfhosted** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Found 3 link issues</td>
    </tr>
  </tbody>
</table>

#### Link issues by [`awesome_bot`](https://github.com/dkhamsing/awesome_bot)

  Line | Status | Link
| ---: | :----: | --- |
2 | Error | https://paste.mills.io <br> SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed
3 | Error | https://url.mills.io <br> SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed
4 | Error | https://wiki.mills.io <br> SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by prologic** [comment]
Self-signed certs :)

File: README.md [hunk 1]
```diff
@@ -889,6 +889,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
   * [mojopaste](http://search.cpan.org/dist/App-mojopaste/) - Perl based pastebin. ([Demo](http://p.thorsen.pm/), [Source Code](https://github.com/jhthorsen/app-mojopaste)) `ARTv2` `Perl`
   * [NoteHub](https://notehub.org) - Free and Hassle-free Pastebin for Markdown Pages. Simple, clean, password provided, generated-short link. ([Demo](https://notehub.org), [Source Code](https://github.com/chmllr/NoteHub)) `MIT` `Nodejs`
   * [Paste](http://phpaste.sourceforge.io/) - Paste is forked from the original source pastebin.com used before it was bought. ([Demo](http://phpaste.sourceforge.net/demo), [Source Code](https://github.com/jordansamuel/PASTE)) `GPLv3` `PHP`
+  * [pastebin](https://github.com/prologic/pastebin) - pastebin is a simple pastebin service with convenient cli supporting ephemeral/temporary storage of pates. ([Demo](https://paste.mills.io) [Source Code](https://github.com/prologic/pastebin/)) `MIT` `Go`
```
Comment (nodiscc) [inline README.md]: s/pates/pastes

Comment (nodiscc) [inline README.md]: Please remove the source code link as it is the same as the main link.

File: README.md [hunk 2]
```diff
@@ -1144,6 +1145,7 @@ See https://staticsitegenerators.net and https://www.staticgen.com
   * [Lstu](https://github.com/ldidry/lstu) - _Let's SHorten That Url_ - Lightweight URL shortener. `WTFPL` `Perl`
   * [Nimbus](https://github.com/ethanal/nimbus) - A URL shortener and file sharer with a drag-and-drop OS X menu bar client and web interface. `MIT` `Python`
   * [Polr](https://project.polr.me/) - A modern, minimalist, modular, and lightweight URL shortener. ([Source Code](https://github.com/Cydrobolt/polr)) `GPLv2` `PHP`
+  * [shorturl](https://github.com/prologic/shorturl) - shorturl is a simple url shortener with very tiny urls. ([Demo](https://url.mills.io) [Source Code](https://github.com/prologic/shorturl/)) `MIT` `Go`
```
Comment (nodiscc) [inline README.md]: * s/url/URL/
 * s/urls/URLs/
 * Please remove the source code link as it is the same as the main link.

File: README.md [hunk 3]
```diff
@@ -1188,6 +1190,7 @@ See also [Documentation Generators](#documentation-generators), [Wikimatrix](htt
   * [TiddlyWiki](http://tiddlywiki.com/) - A reusable non-linear personal web notebook. ([Source Code](https://github.com/Jermolene/TiddlyWiki5)) `BSD` `Nodejs`
   * [Tiki](https://tiki.org) - Wiki CMS Groupware with the most built-in features. ([Demo](https://tiki.org/Demo), [Source Code](https://sourceforge.net/p/tikiwiki/code/HEAD/tree/)) `LGPLv2` `PHP`
   * [TWiki](http://twiki.org/) - TWiki is a Perl-based structured wiki application, typically used to run a collaboration platform, knowledge or document management system, a knowledge base, or team portal. ([Demo](http://twiki.org/cgi-bin/view/Sandbox/WebHome), [Source Code](http://svn.twiki.org/svn/twiki/)) `GPL` `Perl`
+  * [wiki](https://github.com/prologic/wiki) - wiki is a simple markdown based wiki. ([Demo](https://wiki.mills.io) [Source Code](https://github.com/prologic/wiki/)) `MIT` `Go`
```
Comment (nodiscc) [inline README.md]: Please remove the source code link as it is the same as the main link.

**Comment by prologic** [comment]
What the hell?! :) How did I manage to close this PR?

**Comment by prologic** [comment]
Also fixed

**Comment by prologic** [comment]
WoW I managed to kill my locla commit :/

**Comment by Kickball** [comment]
It might be worth opening a new PR with @nodiscc corrections. I can't seem to open from this side.

File: README.md (hunk 2 above)
Comment (Kickball) [inline README.md]: What do you mean by the first two bullet points? @nodiscc

**Comment by prologic** [comment]
Moving to #1105

**Pull request closed without merging**